assets/charts/components/contour/templates/03-envoy.yaml

```diff
@@ -95,11 +95,9 @@ spec:
               fieldPath: metadata.name
         ports:
         - containerPort: 80
-          hostPort: 80
```

Comment:
Can we have tests for it so we don't have a regression again?

Reply:
done